docker/build-tools/docker-entrypoint.sh

```diff
@@ -23,9 +23,16 @@ shopt -s dotglob
 # Make a copy of the hosts's config secrets
 su-exec 0:0 cp -aR /config/* /config-copy/
 
-# Set the ownershp of the host's config secrets to that of the ontainer
+# Set the ownership of the host's config secrets to that of the ontainer
 su-exec 0:0 chown -R "${uid}":"${gid}" /config-copy
 
+# Set the permissions of / to enable writing of /livenessProbe and /healthcheckProbe
+# during test operations
+# TODO(sdake) It would be preferrable to change the path of Galley's /livenessProbe
+# and /healthcheckProbe dynamically during test runs. If that is possible in the future
+# this line may be removed. Currently the information is stored in a structure.
+su-exec 0:0 chmod 777 /
```

Comment:
notice - the ask.ubuntu thread is talking about `-R` - this is sans `-R` so that thread doesn't apply.